sentinelsat/scripts/cli.py

```diff
@@ -250,6 +271,8 @@ def cli(
     search_kwargs = {}
     if sentinel and not (producttype or instrument):
         search_kwargs["platformname"] = "Sentinel-" + sentinel
+    elif gnss:
+        search_kwargs["platformname"] = "Sentinel-" + sentinel
```

Comment:
The `sentinel` parameter might not have been set.

Reply:
done, thanks